### ISSUE TITLE
Actual solar generation time series for Spain 2015 has non-zero loads for every hour of the year

### ISSUE THREAD

**Issue by redekok** [Open-Power-System-Data/time_series#26]
For the actual solar generation time series for Spain 2015 (ES_solar_generation_actual, hourly based time series, from 2015-01-01 to 2015-12-31) there are non-zero loads for every hour of the year. I would expect the loads to be zero between sunset and sunrise.

![image](https://user-images.githubusercontent.com/5946498/81563784-06b39a80-9397-11ea-8d7c-c13cc7b47113.png)

I would be very interested in finding out why this is the case.

**Comment by jgmill** [comment]
That is [concentrated solar power ](https://en.wikipedia.org/wiki/Solar_power_in_Spain#Solar_thermal_power_plants)(CSP).

**Comment by redekok** [comment]
Thanks @jgmill! Just to be sure: there is probably no way to get the separated time series for solar PV on the one hand and CSP on the other hand (through the OPSD or through any other platform)?

**Comment by jgmill** [comment]
Yes there are! I just found the very comprehensive ESIOS data portal from Spanish TSO REE. If you go the [Generation and Consumption](https://www.esios.ree.es/en/generation-and-consumption) section, scroll down to "MORE" and type "solar". You will find `Real time generation` and `Measured generation` for solar PV and thermal separately.

`Measured generation` seems to have publication lag of one month, potentially making it more accurate than `Real time generation`. Note that data from ENTSO-E Transparency that we use for OPSD is real time data.

**Comment by redekok** [comment]
Thanks a lot!

**Comment by redekok** [comment]
Although the issue has yet been closed, I'd like to ask a brief follow-up question.

When I looked at the solar PV profile for 2015, I noticed there are still non-zero loads for every hour of the year, although the nightly values do approach values very close to zero. Could this still be due to some residual production? Or would this have to do with some (minor) measurement errors? 🤔 

**Solar PV**:
![image](https://user-images.githubusercontent.com/5946498/82949589-2dcbb800-9fa4-11ea-9a85-9c36bbb6db3e.png)

![image](https://user-images.githubusercontent.com/5946498/82950040-ef82c880-9fa4-11ea-8ac0-57fea4f183c3.png)
